index.html

```diff
@@ -3,9 +3,33 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./assets/css/styles.css">
   <title>AwesomeBooks</title>
 </head>
+
 <body>
+
+  <h1>Awesome Books</h1>
   
+  <!--<ul> #booklist will hold the list of books-->
+  <ul id="bookslist"></ul>
+  <!--EOF #booklist-->
+
+  <!--form #addbook will be used to add new books-->
+  <form id="addbook" novalidate>
+    <ul>
+      <li>
+        <input type="text" id="title" name="booktitle" placeholder="Title" pattern="((\w|\d)+\s?)*(\w|\d)+" minlength="6" required><br>
+      </li>
+      <li>
+        <input type="text" id="author" name="bookauthor" placeholder="Author" pattern="(\w+\s?)*(\w)+" minlength="6" required><br>
+      </li>
+      <li>
```

Comment:
- [ ] I am not able to add a book because of the `author` name validation please address that
<img width="1440" alt="Screen Shot 2022-12-07 at 23 02 42" src="https://user-images.githubusercontent.com/44745855/206296232-a4567952-1dd2-48ae-89b2-349846b1da5a.png">